.github/actions/install-deps-action/action.yml

```diff
@@ -23,7 +23,12 @@ runs:
         linux-headers-generic linux-tools-common linux-tools-generic make \
         ninja-build pahole pkg-config python3-dev python3-pip python3-requests \
         qemu-kvm rsync stress-ng udev zstd libseccomp-dev libcap-ng-dev \
-        llvm clang python3-full curl meson bpftrace cargo rustc dwarves
+        llvm clang python3-full curl meson bpftrace dwarves rustup
+      shell: bash
+
+    # ensure some toolchain is installed
+    - run: |
+        rustup default nightly
```

Comment:
This was pinned in `lint` before but not in install deps.

Can we do the same tas here so we don't install  a redundant toolchain in this step:

https://github.com/sched-ext/scx/blob/c1690c3dbd93a6aec2c34ad42ea91669e4cc2895/.github/workflows/caching-build.yml#L20

Reply:
kinda prefer default nightly so a default is set for the virtiofsd install later on in the compile kernel path when the cache misses

Reply:
Wouldn't it be more effective to copy our rust-toolchain.toml into that build too? It's not going to be reproducible either way due to the caching, but it's at least then using a toolchain we might have used to build scx.